src/main/java/org/cr22rc/NetworkConfig.java

```diff
@@ -171,7 +171,9 @@ String getName() {
         }
 
         String getMspid() {
```

Comment:
I would be careful here and in other places as I don't think these are optional.  If they're not present probably should throw some type of exeception.

Reply:
If the field is required, should I remove the checking?

Reply:
That's one option. I would think better throwing an exception with an error that the field was missing from the NetworkConfig document.

Reply:
Have you done some testing with this change ?

Reply:
I only do the testing with "affiliation" change from the Jason file. I will leave those fields untouched then